contents/tutorials/bubble-analytics.md

```diff
@@ -0,0 +1,113 @@
+
+![How to add PostHog to Bubble](../images/tutorials/bubble-analytics/adding-posthog.mp4)
+
+PostHog will now begin [automatically capturing](/docs/product-analytics/autocapture) events like pageviews, button clicks, and form inputs on your site.
+
+<ProductScreenshot
+  imageLight={ImgAutocaptureLight} 
+  imageDark={ImgAutocaptureDark} 
+  alt="Autocapture events captured by PostHog in PostHog" 
+  classes="rounded"
+/>
+
+## How to capture custom events
+
+To capture custom events, we create a [Bubble action](https://manual.bubble.io/help-guides/logic/workflows/actions) to run JavaScript code. 
+
+To do so, you first need to install the [Toolbox plugin](https://bubble.io/plugin/toolbox-1488796042609x768734193128308700). Once installed, create a new action by going to the [Workflow tab](https://manual.bubble.io/help-guides/getting-started/navigating-the-bubble-editor/tabs-and-sections/workflow-tab) in Bubble and selecting an event. Then click on **add an action**.
+
+In the menu that appears, click on **Plugins** and then **Run javascript**. This will open a new menu where you can add custom JavaScript code.
+
+![How to add custom javascript in Bubble](../images/tutorials/bubble-analytics/bubble-custom-code.mp4)
+
+To capture PostHog events, add the following code under the **Script** heading:
+
+```js
+window.posthog.capture("your_event_name")
+```
+
+Optionally, you can also include additional information in the event by including parameters:
+
+```js
+posthog.capture('your_event_name', {
+    email: properties.param1,
+})
+```
+
+where `param1` is set in the "Run javascript" menu:
+
+![Set parameters in Bubble's Run Javascript menu](../images/tutorials/bubble-analytics/bubble-toolbox-params.png)
+
+Finally, deploy your changes to live to begin capturing your custom event.
+
+## How to enable session replays
+
+[Session replays](/docs/session-replay) enable you to record and playback user interactions on your site. 
+
+To enable it, go the [Session replay tab](https://us.posthog.com/replay) in PostHog. Then, click on the **Configure** button in the top right and toggle **Record user sessions** in the menu that opens up. 
+
+Once enabled, user sessions will begin to appear in the [Recent recordings tab](https://us.posthog.com/replay/recent).
+
+![How to enable session replays in PostHog](../images/tutorials/bubble-analytics/bubble-record-sessions.mp4)
+
+## Customizing components with feature flags
+
+[Feature flags](/docs/feature-flags) are useful for conditionally showing (or hiding) components based on a rollout percentage and properties. For example, you can use a flag to show or hide a button.
+
+To do this, go to the [feature flags tab](https://us.posthog.com/feature_flags) in PostHog. Click **New feature flag**, enter a key name (for example, `hide-bubble-button`), fill out the details, set release conditions to roll out to 100% of users, and click **Save**.
```

Comment:
We should make the flag `show-bubble-button` so the name matches the state. `hide-bubble-button` is opposite (enabled is showing, disabled is hiding)